nevergrad/functions/control/mujoco.py

```diff
@@ -20,27 +20,49 @@ class GenericMujocoEnv:
         Standard deviation of state values of multiple independent runs.
     num_rollouts: int
         number of independent runs.
+    activation: str:
+        activation function
+    layer_rescaling_coef: float
+        Scaling coefficient of output layers
     random_state: int or None
         random state for reproducibility in Gym environment.
     """
+
     def __init__(self, env_name, state_mean, state_std, num_rollouts,
-                 random_state):
-        self.mean = np.array(state_mean)
-        self.std = np.array(state_std)
+                 activation, layer_rescaling_coef, noise_level, random_state):
+        self.mean = state_mean
+        self.std = state_std
         self.env = gym.make(env_name)
         self.num_rollouts = num_rollouts
-        self.env.seed(random_state)
+        self.random_state = random_state
+        self.activation = activation
+        self.layer_rescaling_coef = layer_rescaling_coef
+        self.noise_level = noise_level
+
+    def _activation(self, x):
+        if self.activation == 'tanh':
+            return np.tanh(x)
+        elif self.activation == "sigmoid":
+            return 1. / (1 + np.exp(-x))
+        else:
+            raise NotImplementedError(r"Activation {self.activation} not implemented.")
 
-    def __call__(self, x):
-        """Compute average cummulative reward of a given policy.
+    def __call__(self, layers):
```

Comment:
Again, there is some weird mismatch about how parameters are handled, can you please crosscheck?

```suggestion
    def __call__(self, *layers: np.ndarray):
```

Reply:
As described above, `layers` is now a tuple of np.ndarray.